src/sentry/api/endpoints/data_export.py

```diff
@@ -6,22 +6,22 @@
 from rest_framework.response import Response
 
 from sentry import features
-from sentry.api.bases.organization import OrganizationEndpoint, OrganizationEventPermission
+from sentry.api.bases.organization import OrganizationEndpoint, OrganizationDataExportPermission
 from sentry.api.serializers import serialize
 from sentry.constants import ExportQueryType
 from sentry.models import ExportedData
 from sentry.tasks.data_export import assemble_download
 
 
 class ExportedDataSerializer(serializers.Serializer):
-    max_value = len(ExportQueryType.as_choices()) - 1
-    query_type = serializers.IntegerField(required=True, min_value=0, max_value=max_value)
-    query_info = serializers.JSONField(required=True)
+
+    query_type = serializers.ChoiceField(choices=ExportQueryType.as_str_choices(), required=True)
     # TODO(Leander): Implement query_info validation with jsonschema
+    query_info = serializers.JSONField(required=True)
```

Comment:
This enforces using strings in requests instead of the integers it was using before

Reply:
(woops, wrong line, it is `query_type` L18 above this.)